bitnami/zookeeper/templates/statefulset.yaml

```diff
@@ -515,16 +510,11 @@ spec:
     {{- if and (not .Values.persistence.dataLogDir.existingClaim) .Values.dataLogDir }}
     - metadata:
         name: data-log
-        annotations:
-        {{- range $key, $value := .Values.persistence.annotations }}
-          {{ $key }}: {{ $value }}
-        {{- end }}
-        {{- if .Values.commonAnnotations }}
-          {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 10 }}
+        {{- if .Values.persistence.annotations }}
+        annotations: {{- include "common.tplvalues.render" (dict "value" .Values.persistence.annotations "context" $) | nindent 10 }}
```

Comment:
Thanks for your contribution @itscaro!

I think that both `commonLabels` and `commonAnnotations` could still coexist with the `persistence.annotations` and `persistence.labels`.

Reply:
But it's not the case with other charts kafka, postgresql.

Reply:
I agree with @itscaro arguments (commonAnnotation may be mutated & consistency with other charts)

Reply:
https://github.com/bitnami/charts/blob/2814eed/bitnami/kafka/templates/statefulset.yaml#L571-L578
https://github.com/bitnami/charts/blob/2814eed/bitnami/postgresql/templates/primary/statefulset.yaml#L611-L618
https://github.com/bitnami/charts/blob/2814eed/bitnami/thanos/templates/storegateway/statefulset-sharded.yaml#L312-L319

etc...

Reply:
I understand the situation. The thing is, at the end of the day neither of both `persistence.annotations` and `commonAnotations` "match the "immutability" of persistence claim" (as helm values are not persistent by nature), but when it gets to its usage, makes sense that in case you need, you use the commonAnnotations dinamically rather than `persistence.annotations`. But even like this, we may have to rely to the nature of how persistence objects are interpreted by kubernetes and if shuold be a way to provide some not inmutable references to a pv/pvc.

Anyway, completely understand now. Approved! :)